clients/gtest/gemm_gtest.cpp

```diff
@@ -44,6 +44,8 @@ Representative sampling is sufficient, endless brute-force sampling is not neces
 const
 vector<vector<int>> matrix_size_range = {
                                         {-1, -1, -1, -1, 1, 1},
+                                        { 3, 33,  3,  3, 3, 3},
```

Comment:
You are correct, {600,500, 500, 500, 600, 500} corresponds to 
M=600, N=500, K=500, lda=500, ldb=600, ldc=500
GEMM requires lda and ldc >= M and ldb >= k.  This example violates lda >= M.  GEMM will return rocblas_status_invalid_size. The argument checks are in line 60 of 
rocBLAS-develop/library/src/blas3/Tensile/gemm.cpp

3, 33, 3, 3, 3, 3  corresponds to    lda=ldb=ldc=3,   and m=3, n=33, k=3.  This satisfies the argument checks lda >=m, ldc >= m, ldb >= k